samples/name_domains_values/Coded_Domain_METOC_Space_Point_Entities.csv

```diff
@@ -1 +0,0 @@
-Name,Value
```

Comment:
@Dbarnes1 - I wonder if these empty METOC space entities domains are what was causing problems with this subtype https://github.com/Esri/military-features-data/issues/153#issuecomment-148850078